samples/graph-activity-feed/csharp/ActivityFeedNotification/Controllers/HomeController.cs

```diff
@@ -169,9 +169,7 @@ public async Task<ActionResult> SendNotificationToGroupChat(TaskInfo taskInfo)
             TaskHelper.AddTaskToFeed(taskInfo);
             var graphClient = SimpleGraphClient.GetGraphClient(taskInfo.access_token);
             var graphClientApp = SimpleGraphClient.GetGraphClientforApp(_configuration["AzureAd:MicrosoftAppId"], _configuration["AzureAd:MicrosoftAppPassword"], _configuration["AzureAd:TenantId"]);
-            var user = await graphClient.Users[taskInfo.userName]
-                      .Request()
-                      .GetAsync();
```

Comment:
Where did we write the logic to get members (Chat, Team)?

Reply:
Not needed it will internally get all users for Chat or for Teams and send feed notification.